src/Core/src/Platform/Standard/ViewExtensions.cs

```diff
@@ -52,7 +52,7 @@ public static partial class ViewExtensions
 
 		internal static Graphics.Rectangle GetNativeViewBounds(this IView view) => view.Frame;
 
-		internal static Matrix4x4 GetViewTransform(this IView view) => new Matrix4x4();
+		internal static Graphics.Matrix4x4 GetViewTransform(this IView view) => new Matrix4x4();
```

Comment:
Isn't this in System.Numerics?

Reply:
It is, https://github.com/drasticactions/maui/blob/dev/timill/adorner-take-1/src/Core/src/Platform/Standard/ViewExtensions.cs#L55

It should be updated to it.